history-of-quakers/modernized/22-chapter-20.adoc

```diff
@@ -1935,11 +2080,16 @@ and the lord Russel, who was made to bow to the axe.
 Now since persecution continued with some color of justice,
 those called Quakers drew up the following address to the king:
 
+[.embedded-content-document.address]
+--
+
+[.letter-heading]
 TO THE KING.
 
 The humble Address of the people commonly called Quakers.
```

Comment:
I didn't know if I should leave this like this or not without any label.

Reply:
yeah i have no idea what to do with that either. let's just leave it i guess.